content/developers/metrics/custom_metrics.md

```diff
@@ -1,6 +1,8 @@
 ---
 title: Custom Metrics
 kind: documentation
+aliases:
+  - /getting_started/custom_metrics
 ---
```

Comment:
Don't you want to add a further reading section on this page ?

Reply:
Yes, but that's not for this PR. This PR is strictly refactoring.

Reply:
👌